data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,420 @@ mission "Remnant: Face to Maw 2B"
 			label end
 			`	"All right, enough history." He turns to the technicians who are finishing the assembly. "The tentacle has been treated to resist decay and the stock mechanisms will provide basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`
 			scene "outfit/void rifle"
+
+
+
+mission "Remnant: Crystal Research 1"
+	name "Retrieve more Crystals"
+	description `Retrieve Crystals from Greenwater, and then Allhome.`
+	minor
+	source "Viminal"
+	destination "Greenwater"
+	cargo "(reserved)" 2
+	to offer
+		random < 30
+		has "Remnant: Learn Sign 2: done"
+		has "Remnant: Technology Available: offered"
+	on offer
+		log "People" "Aeolus" `As an outfitter, Aeolus is responsible for managing the logistics that supply the Remnant with materials, equipment, and resources. In contrast to outfitters in what the Remnant term 'ancestral space,' he is focused on ensuring the efficient optimization of Remnant ships, not making a profit.`
+		conversation
+			`In the spaceport cafeteria you spot the outfitter who asked you to retrieve a shipment of Hai Keystones. He notices you and waves you over, gesturing at an empty chair opposite him.`
+			`	"Greetings, <first>! I trust you are settling in among us?" he signs using simple gestures. "I picked up a rumor that you are learning our language, too." You sign the affirmative as you sit down. "Good, that is useful." He pauses, then continues, "I never introduced myself before. My name is Aeolus, and I help manage resource logistics for our fleets." He eyes you for a moment, then ventures, "You came in here with the appearance of looking for something. Is there something on your mind?"`
+			choice
+				`	(Ask how business is going.)`
+				`	(Ask if there are more jobs like the last one you did for him.)`
```

Comment:
While I don't have a hard rule on when to use brackets and when to do specific replies, I prefer to use brackets for general statements like these so that the player can imagine their own preferred method of talking. Specific replies are best suited to when they include (or are responding to) information that <first> could be reasonably assumed to know but the player may have forgotten.